operator-framework-tools.Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-4.16 AS builder-rhel8
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-4.17 AS builder-rhel8
 ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 WORKDIR /src
 COPY . .
+# Install golang 1.22.2
```

Comment:
/hold
ART is generating a new `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.17` we can use here to coordinate with new go1.22 toolchain.